apps/server/src/infra/etherpad-client/etherpad-client.adapter.ts

```diff
@@ -50,6 +52,29 @@ export class EtherpadClientAdapter {
 		}
 	}
 
+	public async listPadsOfAuthor(userId: EntityId): Promise<PadId[]> {
```

Comment:
Incorrect parameter that expects a authorId

Reply:
The userId !== authorId

Reply:
fixed